SEPADirectDebit/src/test/java/com/braintreepayments/api/SEPADirectDebitClientUnitTest.java

```diff
@@ -540,30 +462,33 @@ public void onBrowserSwitchResult_whenBrowserSwitchStatusSuccess_whenDeepLinkURL
                 .build();
 
         SEPADirectDebitClient sut =
-                new SEPADirectDebitClient(activity, lifecycle, braintreeClient, sepaDirectDebitApi);
-        sut.setListener(listener);
+                new SEPADirectDebitClient(braintreeClient, sepaDirectDebitApi);
+
+        SEPADirectDebitBrowserSwitchResult sepaBrowserSwitchResult = new SEPADirectDebitBrowserSwitchResult(browserSwitchResult);
 
-        sut.onBrowserSwitchResult(activity);
+        sut.onBrowserSwitchResult(sepaBrowserSwitchResult, sepaBrowserSwitchResultCallback);
 
         ArgumentCaptor<Exception> captor = ArgumentCaptor.forClass(Exception.class);
-        verify(listener).onSEPADirectDebitFailure(captor.capture());
+        verify(sepaBrowserSwitchResultCallback).onResult(isNull(), captor.capture());
         verify(braintreeClient).sendAnalyticsEvent("sepa-direct-debit.browser-switch.failure");
 
         Exception exception = captor.getValue();
         assertTrue(exception instanceof BraintreeException);
         assertEquals("Unknown error", exception.getMessage());
     }
-
-    @Test
-    public void getBrowserSwitchResult_getBrowserSwitchResultFromBraintreeClient() {
-        SEPADirectDebitApi sepaDirectDebitApi = new MockSEPADirectDebitApiBuilder().build();
-
-        braintreeClient = new MockBraintreeClientBuilder()
-                .build();
-
-        SEPADirectDebitClient sut =
-                new SEPADirectDebitClient(activity, lifecycle, braintreeClient, sepaDirectDebitApi);
-        sut.getBrowserSwitchResult(activity);
-        verify(braintreeClient).getBrowserSwitchResult(activity);
-    }
-}
\ No newline at end of file
+}
+
+// Save for SEPABrowserSwitchResult_Tests
```

Comment:
Should this be in separate file?

Reply:
I actually think this test is covered [here](https://github.com/braintree/braintree_android/blob/a1b0434bf5b3f4c123fed315201c3e2db2e1c31b/SEPADirectDebit/src/test/java/com/braintreepayments/api/SEPADirectDebitLauncherUnitTest.java#L70-L83). Removed this commented out section in bd911a22fb4783c7388b75b678ffe3d85af69d53